docker/server/build.gradle

```diff
@@ -1,17 +1,25 @@
+import io.deephaven.tools.docker.Architecture
 import io.deephaven.tools.License
 
 plugins {
     id 'com.bmuschko.docker-remote-api'
 }
 
-def baseMap = [
-    'server-base': 'server',
-    'nltk-base': 'server-nltk',
-    'pytorch-base': 'server-pytorch',
-    'sklearn-base': 'server-sklearn',
-    'tensorflow-base': 'server-tensorflow',
+def hostArch = Architecture.fromHost()
```

Comment:
make this configurable from a property so others can test locally (and so github actions can produce images via ci)?